ee/clickhouse/models/group.py

```diff
@@ -2,12 +2,27 @@
 import json
 from typing import Dict, Optional
 
+from django.db.models.signals import post_save
+from django.dispatch.dispatcher import receiver
 from django.utils.timezone import now
 from rest_framework import serializers
 
 from ee.clickhouse.sql.groups import INSERT_GROUP_SQL
 from ee.kafka_client.client import ClickhouseProducer
 from ee.kafka_client.topics import KAFKA_GROUPS
+from posthog.models.group import Group
+from posthog.utils import is_clickhouse_enabled
+
+if is_clickhouse_enabled():
+
+    @receiver(post_save, sender=Group)
+    def person_created(sender, instance: Group, created, **kwargs):
```

Comment:
Nit: Name of the function is wrong.

We want to avoid this pattern if we can - this method won't be called in app code ever and if it is it's guaranteed to be wrong (e.g. properties_last_updated and properties_last_operation are missing), which will cause further bugs down the line.

I assume this is there for tests? If so, worth updating the `create_group` helper instead and perhaps update the name/file location to indicate this is test-specific?

Reply:
Hmm I added the test flag to it. Not quite sure where to move this logic though where it will still be obvious